Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +9,234 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0)
+            {
+                return "N/A.";
+            }
+
+            var splitArray = peopleAndBalances[0].Split(',');
+            var floatArray = ConvertStringArrayToFloat(splitArray[1..]);
+            var names = new[] { splitArray[0] };
+            var heighest = FindHighest(floatArray);
```

Comment:
Highest what? Don't be too lazy to add the extra bits to the method nmes, because it makes the intention more clear, without having to know all the context.

Reply:
Okay, sorry, changed.